pex/pex_builder.py

```diff
@@ -42,6 +42,13 @@
     from typing import Dict, Optional
 
 
+# N.B.: __file__ will be relative when this module is loaded from a "" `sys.path` entry under
```

Comment:
This took me a good bit to understand and figure out but hopefully is clear now with this encoding of the fix.

The PR should be all good to review / go green now.